web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -346,104 +348,103 @@ GROUP BY 1`;
     this.datasourceQueryManager.terminate();
   }
 
-  renderDropDataAction() {
-    const { dropDataDatasource } = this.state;
-    if (!dropDataDatasource) return;
+  renderUnuseAction() {
+    const { dataSourceToMarkAsUnusedAllSegmentsIn } = this.state;
+    if (!dataSourceToMarkAsUnusedAllSegmentsIn) return;
 
     return (
       <AsyncActionDialog
         action={async () => {
           const resp = await axios.delete(
-            `/druid/coordinator/v1/datasources/${dropDataDatasource}`,
+            `/druid/coordinator/v1/datasources/${dataSourceToMarkAsUnusedAllSegmentsIn}`,
             {},
           );
           return resp.data;
         }}
-        confirmButtonText="Drop data"
-        successText="Data drop request acknowledged, next time the coordinator runs data will be dropped"
-        failText="Could not drop data"
+        confirmButtonText="Mark as unused all segments in data source"
```

Comment:
This is way too long to be a button. Maybe just call it "Mark as unused" and make sure to specify what is going on in the text of the dialog (which you are mostly doing already)

Reply:
I shortened confirm button texts but just removed "in data source" parts, IMO even shorter names are too ambiguous.